spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/BigQueryStrategy.scala

```diff
@@ -134,7 +136,7 @@ class BigQueryStrategy(expressionConverter: SparkExpressionConverter, expression
    * @return An object of type Option[BQSQLQuery], which is None if the plan contains an
    *         unsupported node type.
    */
-  def generateQueryFromPlan(plan: LogicalPlan): Option[BigQuerySQLQuery] = {
+  def generateQueryFromCleanedPlan(plan: LogicalPlan): Option[BigQuerySQLQuery] = {
```

Comment:
Same comment as in PR#705. 
Just a thought: can the name instead be `cleanUpAndGenerateQueryFromPlan`, since we are calling generateQueryFromCleanedPlan recursively later and would be better to call generateQueryFromPlan instead

Reply:
Changed the wrapper method to `generateQueryFromOriginalLogicalPlan` and kept `generateQueryFromPlan` as is